src/swagger/get.ts

```diff
@@ -44,6 +45,7 @@ export const generateSwagger = (
       ...(summary && { summary }),
       ...(description && { description }),
       ...(externalDocs && { externalDocs }),
+      ...(security && { security: getSecurity(security) }),
```

Comment:
One issue here is that there is no check that the security IDs passed here are defined in the root level `securityDefinitions` which they would need to be for swagger to work. This does make the "flexibility" portion added in this PR a little bit risky since it allows user-error. 

Maybe there is a runtime check that can be _cleanly_ added? Or maybe I should rollback the flexibility and define a few allowed security IDs that can be used...

Leaving as it so for now since this works and isn't very challenging to resolve, I am helping to enforce this by using a new `SecurityDefinitionId` enum introduced in the related [bonsai repo PR](https://github.com/ShopBonsai/bonsai/pull/14961)